libvirtd-desktop/README.md

```diff
@@ -7,13 +7,43 @@
   ```
   $ sudo systemd-sysusers /usr/lib/sysusers.d/libvirt-qemu.conf
   ```
+- Create the `libvirtdbus` user and group:
+  ```
+  $ sudo bash -c 'getent group libvirtdbus >/dev/null || groupadd -r libvirtdbus'
+  $ sudo bash -c 'getent passwd libvirtdbus >/dev/null || \
+      useradd -r -g libvirtdbus -d / -s /sbin/nologin \
+      -c "Libvirt D-Bus bridge" libvirtdbus'
+  ```
+- Create the `libvirt` group to allow password-less polkit access to libvirt deamons:
+  ```
+  $ sudo bash -c 'getent group libvirt >/dev/null || groupadd -r libvirt'
+  ```
+- Optional: Add your UID to the libvirt group
+  ```
+  $ sudo usermod -G -a libvirt $YOUR-UID-HERE
+  ```
+- Create the /etc/dbus-1/system.d/org.libvirt.conf file
```

Comment:
Hum, can't we copy this one form `/usr/etc` isntead?

Reply:
Good catch!  For whatever reason, I completely missed that config file. I've updated the doc to copy it into the correct location and fix the SELinux label.